pkg/poolcache/poolcache.go

```diff
@@ -89,6 +90,19 @@ func (c *Cache) service() {
 		req := <-c.requestChannel
 		resp := &response{}
 		switch req.requestType {
+		case getValueByFunctionAndAndress:
+			otelUtils.LoggerWithTraceID(req.ctx, c.logger).Debug("Get value By Function And Address", zap.String("function", req.function.(string)), zap.String("address", req.address.(string)))
+
+			value := c.cache[req.function][req.address]
+
+			if value != nil {
+				otelUtils.LoggerWithTraceID(req.ctx, c.logger).Debug("Found function service", zap.String("function", req.function.(string)), zap.String("address", req.address.(string)))
+				resp.value = value.val
+			} else {
+				otelUtils.LoggerWithTraceID(req.ctx, c.logger).Warn("Address or Function was not found", zap.String("function", req.function.(string)), zap.String("address", req.address.(string)))
```

Comment:
Same issue with letter casing here

Reply:
Handled the letter casing. Please take a look